src/store/create-state.js

```diff
@@ -13,9 +15,13 @@ function createStoreState(key, initialState) {
     },
     setState(nextState) {
       currentState = nextState;
-      // Instead of notifying all handlers immediately, we wait next tick
-      // so multiple actions affecting the same store gets combined
-      schedule(storeState.notify);
+      if (defaults.unstable_concurrent && unstable_concurrent !== false) {
+        startTransition(storeState.notify);
```

Comment:
I am not sure you are "allowed" to `startTransition` from within library. This should be user-space controlled.